recipes/ipython-sql/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.3.6" %}
+{% set md5 = "d4feb00ac5806d7640b2545a43974766" %}
+
+package:
+  name: ipython-sql
+  version: {{ version }}
+
+source:
+  fn: ipython-sql-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/source/i/ipython-sql/ipython-sql-{{ version }}.tar.gz
+  md5: {{ md5 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - prettytable
+    - ipython >=1.0  # [not win]
+    - ipython >=1.0  # [win and not py35]
+    - ipython >=1.0,<5  # [win and py35]
+    - sqlparse
+    - six
+
+  run:
+    - python
+    - prettytable
+    - ipython >=1.0  # [not win]
+    - ipython >=1.0  # [win and not py35]
+    - ipython >=1.0,<5  # [win and py35]
+    - sqlalchemy >=0.6.7
+    - sqlparse
+    - six
+
+test:
+  imports:
+    - sql
+
+about:
+  home: https://pypi.python.org/pypi/ipython-sql
+  license: MIT
```

Comment:
Ideally we would include the license file. However, they don't provide one. 😞

Have raised issue ( https://github.com/catherinedevlin/ipython-sql/issues/58 ) to track this. Could we please add a comment here to add the [`license_file`]( http://conda.pydata.org/docs/building/meta-yaml.html#license-file ) once they have provided one?